### PR TITLE
[REGISTRAR] update inter-account domain move instructions

### PR DESCRIPTION
**Pull request by allysmith424**
### Summary

This PR updates the inter-account domain registration move instructions to include a description of the self-serve option.

### Documentation checklist

- [X] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 1]
```diff
@@ -19,12 +19,40 @@ Before proceeding, please be aware of the following:
 
 ## 1. Prepare for the transfer
```
Comment (dcpena) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]: If we're going to refer to the transfer as a move in the sentence after this, should we change the header to say "move?"

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 2]
```diff
@@ -19,12 +19,40 @@ Before proceeding, please be aware of the following:
...
 
-Before you request the transfer, you need to [disable DNSSEC](/dns/dnssec/#disable-dnssec) for the domain(s) and ensure the domain(s) are set up and ready in the new dashboard account you intend to transfer them to.
+Before you request the move, you will need to do the following:
 
-## 2. Submit a transfer request
+* Obtain the [account ID](/fundamentals/setup/find-account-and-zone-ids) of the new account
+* Add the domain as a website to the new account and select a plan
+* [disable DNSSEC](/dns/dnssec/#disable-dnssec) for the domain(s) and ensure the domain(s) are set up and ready in the new dashboard account you intend to transfer them to.
```
Comment (dcpena) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
* [Disable DNSSEC](/dns/dnssec/#disable-dnssec) for the domain(s) and ensure the domain(s) are set up and ready in the new dashboard account you intend to transfer them to.
```

Comment (dcpena) [reply src/content/docs/registrar/account-options/inter-account-transfer.mdx]: For "domain(s)," choose either singular or plural but let's delete the parentheses.

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 3]
```diff
@@ -19,12 +19,40 @@ Before proceeding, please be aware of the following:
...
 
-Go to the Support and select **Contact Cloudflare** from the Cloudflare dashboard. Select the category **Registrar** and then **Transfer a domain between Cloudflare accounts**. Select the domain(s) in your account that you wish to transfer, enter a Summary and select **Open a Ticket**. 
+The following pre-conditions must be met before the domain can be moved:
 
-The form will ask you to confirm that you understand the process and ask you to specify an email address of a new superadmin for the account you intend to transfer the domain(s) to. The new super admin you specify will need to reply to the support ticket to agree to receive the transfer.
+* The domain must be added to the new account as a website and a plan must be selected.
+* The domain must not be administratively locked, such as being locked due to a dispute or court order.
+* The domain must not have any of the following registry statuses: pendingDelete, redemptionPeriod, pendingTransfer.
```
Comment (dcpena) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
* The domain must not have any of the following registry statuses: `pendingDelete`, `redemptionPeriod`, or `pendingTransfer`.
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 4]
```diff
@@ -19,12 +19,40 @@ Before proceeding, please be aware of the following:
...
+* The registrant email address must be verified.
+* There must not be a pending Change of Registrant request. If there is a pending request it should be completed before initiating the move request.
```
Comment (dcpena) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
* A pending Change of Registrant request cannot be present. If there is a pending request, it should be completed before initiating the move request.
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 5]
```diff
@@ -19,12 +19,40 @@ Before proceeding, please be aware of the following:
...
+* DNSSEC must be turned off. It can be re-enabled on the new zone once the move completes.
+* If the current zone is locked, the lock must be released.
+
+## 2. Submit the move request
+
+There are two ways to submit a move request. The first option is to submit a request via a support ticket following the procedure described below. The second option is to use the self-serve feature in the dashboard.
+
+**The self-serve option is currently in beta and only accessible to a limited number of customers. If the option is not currently available in your account, please submit via a support ticket.**
```
Comment (dcpena) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
:::note
The self-serve option is currently in beta and only accessible to a limited number of customers. If the option is not currently available in your account, submit a request through a support ticket.
:::
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 6]
```diff
@@ -19,12 +19,40 @@ Before proceeding, please be aware of the following:
...
+
+### Submitting via a support ticket:
```
Comment (dcpena) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
### Submit via a support ticket
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 7]
```diff
@@ -19,12 +19,40 @@ Before proceeding, please be aware of the following:
...
+
+Go to Support and select **Contact Cloudflare** from the Cloudflare dashboard. Select the category **Registrar** and then **Transfer a domain between Cloudflare accounts**. Select the domain(s) in your account that you wish to transfer, enter a Summary and select **Open a Ticket**.
```
Comment (dcpena) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
1. Go to Support and select **Contact Cloudflare** from the Cloudflare dashboard. 
2. Select the category **Registrar** and then **Transfer a domain between Cloudflare accounts**. 
3. Select the domain in your account that you wish to transfer.
4. Enter a summary and select **Open a Ticket**.
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 8]
```diff
@@ -19,12 +19,40 @@ Before proceeding, please be aware of the following:
...
+
+The form will ask you to confirm that you understand the process and ask you to specify an email address of a new super-admin for the account you intend to transfer the domain(s) to. The new super admin you specify will need to reply to the support ticket to agree to receive the transfer.
```
Comment (dcpena) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
The form will ask you to confirm that you understand the process and ask you to specify an email address of a new super admin for the account you intend to transfer the domain to. The new super admin you specify will need to reply to the support ticket to agree to receive the transfer.
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 9]
```diff
@@ -19,12 +19,40 @@ Before proceeding, please be aware of the following:
...
 
 Once this has been done, the transfer takes around three to five working days to complete.
+
+### Submitting using the self serve option (if available):
```
Comment (dcpena) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
### Submit using the self serve option (if available)
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 10]
```diff
@@ -19,12 +19,40 @@ Before proceeding, please be aware of the following:
...
+
+The self-serve option will appear under the Configuration tab of the Manage Domain page. If the option is available, begin the submission process by clicking the Start button and follow the instructions.
```
Comment (dcpena) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
The self-serve option will appear under the **Configuration** tab of the **Manage Domain** page. If the option is available, begin the submission process by selecting the **Start** button and follow the instructions.
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 11]
```diff
@@ -19,12 +19,40 @@ Before proceeding, please be aware of the following:
...
+
+**Important**: Please be sure to review the pre-conditions described above. If those conditions have not been met the domain move will not be completed.
```
Comment (dcpena) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
**Important**: Review the pre-conditions described above. If those conditions have not been met, the domain move will not be completed.
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 12]
```diff
@@ -19,12 +19,40 @@ Before proceeding, please be aware of the following:
...
+
+Once the move request has been submitted, the gaining account will receive an email notifying them of the request and providing instructions for how to approve the request. The gaining account must log into their account, navigate the Manage Domains (under Domain Registration). At the top of the page a message will appear stating that there are domains requiring action to be taken. Clicking View Actions will display the domains pending move along with buttons to accept or reject the request. Action must be taken within 5 days of the request. If no action is taken within the 5 days, the request will be automatically canceled.
```
Comment (dcpena) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
Once the move request has been submitted, the gaining account will receive an email notifying them of the request and will provide instructions for how to approve the request. 

The gaining account must log into their account and go to **Manage Domains** (under Domain Registration). A message will appear at the top of the page stating that there are domains requiring action to be taken. 

Select **View Actions** to display the domains with a pending move along and choose to accept or reject the request. Action must be taken within five days of the request. 

If no action is taken within the five days, the request will be automatically canceled.
```

**Comment by allysmith424** [comment]
@dcpena thank you for your review -- I just made all changes you requested, if you'd like to have another look

**Review by dcpena — APPROVED** [review]

**Pull request merged**